frontend/src/components/configmap/Details.tsx

```diff
@@ -33,6 +34,7 @@ export default function ConfigDetails() {
           ))
         )}
       </SectionBox>
+      <EventsSection />
```

Comment:
The events we want to show in the resource details view is the ones related to that resource alone (as #911 describes).
We already have a [component](https://github.com/headlamp-k8s/headlamp/blob/main/frontend/src/components/common/ObjectEventList.tsx) that does this for [HPA](https://github.com/headlamp-k8s/headlamp/blob/main/frontend/src/components/horizontalPodAutoscaler/Details.tsx) and a few other details view.

So we can either add it to every view, or maybe a better idea is to have it be added to the DetailsGrid after DetailsViewSection (if a prop `withEvents` is true, as by default it should be false so we don't break the use of DetailsGrid by plugins).

Reply:
updated